cloudevents/adapters/github.md

```diff
@@ -595,6 +925,36 @@ based on the specified event.
 | `time`                | "security_advisory.updated_at" value             |
 | `data`                | Content of HTTP request body                     |
 
+### SecurityAndAnalysisEvent
+
+| CloudEvents Attribute | Value                                 |
+| :-------------------- | :------------------------------------ |
+| `id`                  | "X-GitHub-Delivery" HTTP header value |
+| `source`              | "repository.url" value                |
+| `specversion`         | `1.0`                                 |
+| `type`                | `com.github.security_and_analysis`    |
+| `datacontentencoding` | Omit                                  |
+| `datacontenttype`     | `application/json`                    |
+| `dataschema`          | Omit                                  |
+| `subject`             | Omit                                  |
```

Comment:
maybe repository.id ?

Reply:
Same as first comment, we have the repository url in source, not sure we need the id in subject